lib/instrumenter.js

```diff
@@ -442,7 +443,7 @@
         instrumentSync: function (code, filename, babelConfig) {
             var program, transformed;
 
-            babelConfig = babelConfig || {};
+            babelConfig = Object.assign({}, this.opts.babelConfig, babelConfig);
```

Comment:
`Object.assign` is an es6 method. We should use something with wider support here.